README.md

```diff
@@ -0,0 +1,38 @@
+# react-date-time-range
+
+[React](https://facebook.github.io/react/) Component implementing a range by wrapping two [date-time-group](https://github.com/holidayextras/react-date-time-group)s.
+
+```
+var DateTimeRange = require('react-date-time-range');
+var DateTimeGroup = require('react-date-time-group');
+
+React.render((
+  <DateTimeRange onChange={console.log.bind(console)}>
+    <DateTimeGroup />
```

Comment:
If connecting two DateTimeGroup components together is the function of this component then should we have to pass them as children? What if we pass only 1 or if we pass 3?